ui-library-transfer/README.md

```diff
@@ -0,0 +1,40 @@
+---
+page_type: sample
+languages:
+- javascript
+products:
+- azure
+- azure-communication-services
+---
+
+# Get Started with Composites - Handle transfer request
+
+This sample showcases how Call Composites can be used to handle transfer requests. The current beta version can only
+receive transfer requests from Teams users in 1 on 1 calls.
+
+## Prerequisites
+
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F)  .
+- [Node.js](https://nodejs.org/en/) Active LTS and Maintenance LTS versions (8.11.1 and 10.14.1 recommended).
+- An active Communication Services resource. [Create a Communication Services resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource). You will need the endpoint value for the resource
+- An identity with VoIP. Generate an identity using the [Azure Portal](https://docs.microsoft.com/azure/communication-services/quickstarts/identity/quick-create-identity).
+- A phone number procured through Azure Communication Service portal using [Azure Portal](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/telephony/get-phone-number).
```

Comment:
Should this be updated about the teams user and some tenant information?

Reply:
Thanks. Updated.